src/widgets/FridgeHeader.js

```diff
@@ -2,24 +2,50 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { connect } from 'react-redux';
-import { StyleSheet, Text } from 'react-native';
+import { StyleSheet } from 'react-native';
 import { TextWithIcon } from './Typography/index';
 import { BatteryIcon, CogIcon, DownloadIcon } from './icons';
-import { APP_FONT_FAMILY, BLACK, DARKER_GREY, MISTY_CHARCOAL } from '../globalStyles/index';
+import {
+  APP_FONT_FAMILY,
+  BLACK,
+  DARKER_GREY,
+  FINALISE_GREEN,
+  MISTY_CHARCOAL,
+} from '../globalStyles/index';
 import { IconButton } from './IconButton';
-import { FlexView } from './FlexView';
 import { selectSensorState } from '../selectors/Entities/sensor';
 import { gotoEditSensorPage } from '../navigation/actions';
 import { LastSensorDownload } from './LastSensorDownload';
 import { BlinkSensorButton } from './BlinkSensorButton';
+import { Circle } from './Circle';
+import { FlexRow } from './FlexRow';
 
 const formatBatteryLevel = batteryLevel => `${batteryLevel}%`;
 
-export const FridgeHeaderComponent = ({ batteryLevel, name, macAddress, editSensor, showCog }) => (
+export const FridgeHeaderComponent = ({
+  batteryLevel,
+  name,
+  macAddress,
+  editSensor,
+  showCog,
+  showTitle,
+}) => (
   <>
-    <FlexView justifyContent="center">
-      <Text style={localStyles.paperTitleText}>{name}</Text>
-    </FlexView>
+    <FlexRow flex={1} alignItems="center">
+      {showTitle && (
```

Comment:
was sort of trying to avoid this, but it looks a bit silly on the `SensorEditPage` where the `Sensor Name` editing input is directly under the title!